core/identity-management/iin-agent/src/fabric-ledger/connector.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Contract } from 'fabric-network';
+import { LedgerBase } from '../common/ledgerBase';
+import { walletSetup } from './walletUtils';
+import { getAllMSPConfigurations, invokeFabricChaincode, queryFabricChaincode } from './networkUtils';
+import * as path from 'path';
+import * as fs from 'fs';
+
+export class FabricConnector extends LedgerBase {
+    connectionProfilePath: string;
+    configFilePath: string;
+    networkId: string;
+    orgMspId: string;
+    walletPath: string;
+
+    constructor(ledgerId: string, contractId: string, connectionProfilePath: string, configFilePath: string, walletPath: string) {
+        super(ledgerId, contractId);
+        this.connectionProfilePath = connectionProfilePath ? connectionProfilePath : path.resolve(__dirname, './', 'connection_profile.json');
+        this.configFilePath = configFilePath ? configFilePath : path.resolve(__dirname, './', 'config.json');
+        this.networkId = process.env.NETWORK_NAME ? process.env.NETWORK_NAME : 'network1';
```

Comment:
Should you take this networkId also as (optional) parameter, as all other variables are  taken as param instead of env variable?